app/src/main/java/ar/com/wolox/android/example/ui/home/news/NewsHolder.kt

```diff
@@ -0,0 +1,29 @@
+package ar.com.wolox.android.example.ui.home.news
+
+import android.support.v7.widget.RecyclerView
+import android.view.View
+import ar.com.wolox.android.R
+import ar.com.wolox.android.example.model.New
+import kotlinx.android.synthetic.main.item_new.view.*
+import org.joda.time.format.DateTimeFormat
+import org.ocpsoft.prettytime.PrettyTime
+
+class NewsHolder(view: View) : RecyclerView.ViewHolder(view) {
+
+    fun bindHolder(item: New) {
+        itemView.mNewTitle.text = item.title
+        itemView.mNewText.text = item.text
+
+        val localDate: java.util.Date = DateTimeFormat
+                .forPattern("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'")
+                .withLocale(java.util.Locale.getDefault())
```

Comment:
I think that to make a cleaner code, you should import the `java.util.Locale` library and use `Locale.getDefault()` here.